### DISCUSSION TITLE
Center the annotation and Var ground_station_location isn't visible on image

### DISCUSSION THREAD

**Discussion by rbender23** [jekhokie/raspberry-noaa-v2#430] [Q&A] [answered]
Hi there!

Just updated to v2. Works fine over all.  But have two questions: How can I really center the annotation on every image. 
Every annotation is on the left, but I like to have it centered.

This is my code:
`               <p style="color: yellow; font-weight: bold;text-align: center;">
                  {{ sat_name }} | {{ enhancement }} | {{ capture_start }} UTC | Max Elev: {{ sat_max_elevation }}&#176; | {{ pass_side }}
                  Sun Elevation: {{ sun_elevation }}&#176; | {{ pass_direction }}<br>
                  Ground Station:  {{ ground_station_location }}
`
Second question: Any hints if the variable ground_station_location isn't visible on the image?
I tried in the conf: 
`GROUND_STATION_LOCATION='my text' `
and 
`GROUND_STATION_LOCATION=my text `

Tnx for helping out

**Comment by dom-robinson** [comment, marked as answer]
In settings.yml try 

``` image_annotation_location: 'Centre' ```

and for Ground Station try

``` ground_station_location: 'my text' ```

hope that helps!! ( i think its the '=' signs that were your issue)..

Dom

**Reply by rbender23** [reply]
Thanks for your help.
In the settings.yml you are right with '=' signs as an issue. But in the conf-File, as I wrote, it must be with '='.

My first question with centering the annotation is solved with 'IMAGE_ANNOTATION_LOCATION=North' in the conf-File.
